packages/forma-36-tokens/src/tokens/typography/line-height.js

```diff
@@ -1,6 +1,13 @@
 const lineHeight = {
   'line-height-default': '1.5',
   'line-height-condensed': '1.25',
+  'line-height-4xl': '3.5rem',
```

Comment:
I see no reason to have more than two different line-heights. We're introducing unnecessary complexity.

Heading and Display should use the condensed one (1.25).
All the rest should use the default one (1.5).